chainntnfs/bitcoindnotify/bitcoind_test.go

```diff
@@ -105,12 +105,12 @@ func syncNotifierWithMiner(t *testing.T, notifier *BitcoindNotifier,
 // TestHistoricalConfDetailsTxIndex ensures that we correctly retrieve
 // historical confirmation details using the backend node's txindex.
 func TestHistoricalConfDetailsTxIndex(t *testing.T) {
```

Comment:
We might as well change the functions' names and docs.

Reply:
what would you change it to? I think the func name is still accurate. it is just that the subtests within the func were named incorrectly

Reply:
Oh my bad, somehow read them differently😂